### PR TITLE
Update ES to 5.x

### PR DESCRIPTION
**Pull request by mburke5678**
**Productize Upstream Curator**
https://trello.com/c/0VsVlN5W/676-3-productize-upstream-curator-52-epic-ois-es5 -- changes in this PR
**Failed to show Kibana when redirect from console**  
https://bugzilla.redhat.com/show_bug.cgi?id=1614255 -- changes in this PR
**Need new dist-gits for new images for OCP 3.10 for elasticsearch 5**
https://projects.engineering.redhat.com/browse/RCM-30934 -- Changes in this PR
**Productize ElasticSearch**
https://trello.com/c/tugijL9G/683-8-productize-elasticsearch-epic-ois-es5 -- no changes based on Trello
**Implement Single Search Guard Index**
https://trello.com/c/d3Fadd6D/557-3-implement-single-searchguard-index-epic-ois-es5 -- no changes based on Trello

### CODE REVIEW

**Comment by mburke5678** [comment]
Commit e1fda52968c10bfc023baa0bd568564c4b38f0a4 is based on https://trello.com/c/0VsVlN5W/676-3-productize-upstream-curator-52-epic-ois-es5
@josefkarasek Can you review this commit for me?

**Comment by mburke5678** [comment]
Commit 59aebe08612689432df68303a3b12678125bc33c is based on Jira:
Need new dist-gits for new images for OCP 3.10 for elasticsearch 5
https://projects.engineering.redhat.com/browse/RCM-30934 -

**Comment by mburke5678** [comment]
Commit 98fa96f8aef569a8919c1cd3e91a291ec5e8b750 based on BZ https://bugzilla.redhat.com/show_bug.cgi?id=1614255 
@josefkarasek Can you review this commit for me?

**Comment by mburke5678** [comment]
Per email from @josefkarasek 
Commit https://github.com/openshift/openshift-docs/commit/a8a5f6e8c9fba730044b46fba47a5ae38802be17
"yes, it looks good"

**Comment by mburke5678** [comment]
@josefkarasek @jcantrill 
I added the following to the release notes:
_Due to a change in the authentication for the Kibana web console, you will need to log back into the console after upgrade and every 168 hours after initial log in. The Kibana console was migrated to oauth-proxy._
Please let me know if this is correct.

**Comment by mburke5678** [comment]
@openshift/sig-logging Team, this is the PR for the elasticsearch upgrade to 5 for docs. is this the right list?

**Comment by jcantrill** [comment]
@mburke5678 yes

**Comment by mburke5678** [comment]
@jcantrill Can you ask someone to review these two commits for the 3.11 docs?
**Productize Upstream Trello**
https://trello.com/c/0VsVlN5W/676-3-productize-upstream-curator-52-epic-ois-es5
**Need new dist-gits for new images for OCP 3.10 for elasticsearch 5**
https://projects.engineering.redhat.com/browse/RCM-30934

File: install/disconnected_install.adoc
Comment (ewolinetz) [inline install/disconnected_install.adoc]: We no longer use the `logging-auth-proxy`

**Review by ewolinetz — CHANGES_REQUESTED** [review]

File: install/disconnected_install.adoc
Comment (mburke5678) [reply install/disconnected_install.adoc]: @ewolinetz Is the change to drop `logging-auth-proxy` in 3.11+?

**Review by jcantrill — APPROVED** [review]

**Comment by mburke5678** [comment]
@ewolinetz Do the other changes around the new images look right? 
Does the user add a _5_ to the image name or `<latest>`?
$ docker pull registry.access.redhat.com/openshift3/logging-kibana5
$ docker pull registry.access.redhat.com/openshift3/logging-kibana<latest>

**Comment by mburke5678** [comment]
@QiaolingTang PTAL
**Productize Upstream Trello**
Commit e1fda52968c10bfc023baa0bd568564c4b38f0a4 is based on https://trello.com/c/0VsVlN5W/676-3-productize-upstream-curator-52-epic-ois-es5

 @anpingli PTAL
**Failed to show Kibana when redirect from console**
Commit 98fa96f8aef569a8919c1cd3e91a291ec5e8b750 based on BZ https://bugzilla.redhat.com/show_bug.cgi?id=1614255

**Comment by ewolinetz** [comment]
@mburke5678 yes, the other images look correct. 
They would use `$ docker pull registry.access.redhat.com/openshift3/logging-kibana5`

File: install/disconnected_install.adoc
Comment (anpingli) [inline install/disconnected_install.adoc]: logging-auth-proxy  is replaced with oauth-proxy.

File: install_config/aggregate_logging.adoc
Comment (anpingli) [inline install_config/aggregate_logging.adoc]: 168 hours?  The OCP valid  session will expire in 24 hours.  I guess the Kibana Login cookie should be same. Let me have a try.

File: install/disconnected_install.adoc
Comment (anpingli) [reply install/disconnected_install.adoc]: @mburke5678  Shall we replace the registry.access.redhat.com with registry.redhat.io now?

File: install_config/aggregate_logging.adoc
Comment (anpingli) [reply install_config/aggregate_logging.adoc]: You need to log in to the kibana console when your valid OCP token expired.  the default is 24 hours.

File: install/disconnected_install.adoc
Comment (mburke5678) [reply install/disconnected_install.adoc]: @anpingli As far as I know, we are not changing to registry.redhat.io at this time.

File: install_config/aggregate_logging.adoc
Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: @josefkarasek Can you respond on Anping's comment on 168 hours? In an email, you suggested "the log in expiration period is 1 week (168hours) "

File: install/disconnected_install.adoc
Comment (anpingli) [reply install/disconnected_install.adoc]: I mean  "docker pull registry.access.redhat.com/openshift3/logging-kibana5:<tag>"

Comment (mburke5678) [reply install/disconnected_install.adoc]: @ewolinetz Can you comment on Anping's request to add the `<tag>` to these images, which would add the OpenShift versioning to the command?

Comment (ewolinetz) [reply install/disconnected_install.adoc]: @mburke5678 there was `:<tag>` before, and the other images still have it on them. I think we should keep tag on the image. Only the image names should have change (added `5`)

Comment (mburke5678) [reply install/disconnected_install.adoc]: @ewolinetz @anpingli  Thanks!

File: install/disconnected_install.adoc
Comment (mburke5678) [reply install/disconnected_install.adoc]: Fixed. Not sure how that happened! Thanks!

**Comment by anpingli** [comment]
@mburke5678  Could you confirm again about the default registry?   The default registry is registry.redhat.io in openshift-ansible v3.11 playbooks .

**Comment by mburke5678** [comment]
@anpingli I believe I gave you incorrect information before. Someone is going to update the docs from registry.access.redhat.com to  registry.redhat.io. 
@vikram-redhat can you confirm?

**Comment by vikram-redhat** [comment]
@mburke5678 The default registry is registry.redhat.io for 3.11. @bmcelvee is indeed working on a mass change for this here: https://github.com/openshift/openshift-docs/pull/12086 (and I see that she does have the disconnected_install as a file that is being changed).

We will have to be careful here when merging, as I expect a merge conflict because of this. I suggest @mburke5678 you merge your changes first and @bmcelvee merges next after doing a rebase and updating her PR.

Also tagging @kalexand-rh as she is working on the disconnected_install for awareness.

File: install_config/aggregate_logging.adoc
Comment (josefkarasek) [reply install_config/aggregate_logging.adoc]: @anpingli good point

@jcantrill we don't overwrite that, maybe we should openshift/openshift-ansible/pull/10160

**Comment by kalexand-rh** [comment]
This will probably conflict with https://github.com/openshift/openshift-docs/pull/12024/files. I'll try to hold that one until this one merges.

**Comment by jcantrill** [comment]
Still looks find for me

**Comment by anpingli** [comment]
@mburke5678  Could you modify the sentence  "You need to log in to the Kibana console when your valid login cookie expires after 168 hours (1 week)".  A quick fix is to drop the words "after 168 hours (1 week)".

**Comment by mburke5678** [comment]
@anpingli Thank you for suggestion above. I removed the reference to 168 hours until @josefkarasek PR is merged.

**Comment by mburke5678** [comment]
I will merge this PR unless anyone has objections.

**Comment by anpingli** [comment]
LGTM

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Comment by vikram-redhat** [comment]
@mburke5678 go ahead and merge and CP so other PRs that are dependent can be worked on. If there are still any changes, we will need to add them in followups.

**Comment by vikram-redhat** [comment]
@mburke5678 - cped to 3.11?

**Comment by mburke5678** [comment]
@kalexand-rh Feel free to merge https://github.com/openshift/openshift-docs/pull/12024/files

**Pull request merged**
